floyd/cli/experiment.py

```diff
@@ -161,32 +160,19 @@ def logs(id, url, tail, sleep_duration=1):
 
 @click.command()
 @click.option('-u', '--url', is_flag=True, default=False, help='Only print url for accessing logs')
-@click.option('-d', '--download', is_flag=True, default=False,
```

Comment:
I wonder if this will break the workflow of any of our current users. May be we should add this functionality back.

Reply:
We don't have to hold the release for this. We can do this as a separate commit.

Reply:
Yeah, let me add it to our queue.